src/api/app.ts

```diff
@@ -72,24 +57,7 @@ initApp()
 
     useExpressServer(app, {
       routePrefix: "/1.0",
-      controllers: [
-        ApiKeyController,
-        AuthController,
-        CalloutController,
-        CalloutResponseController,
-        CalloutResponseCommentController,
-        ContentController,
-        EmailController,
-        ContactController,
-        NoticeController,
-        PaymentController,
-        SegmentController,
-        SignupController,
-        StatsController,
-        ResetPasswordController,
-        ResetDeviceController,
-        UploadController
-      ],
+      controllers,
```

Comment:
Is this working? Otherwise you could do this: `controllers: Object.values(controllers)`.

Reply:
`./controllers/index.ts` is exporting an array so it works already :)